.github/workflows/kb_sdk_test.yaml

```diff
@@ -0,0 +1,54 @@
+name: KBase SDK Tests
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+
+  sdk_tests:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out GitHub repo
+      if: "!contains(github.event.head_commit.message, 'skip ci')"
+      uses: actions/checkout@v2
+
+    - name: Check out Actions CI files
+      if: "!contains(github.event.head_commit.message, 'skip ci')"
+      uses: actions/checkout@v2
```

Comment:
is this a duplicate of the above? If not, what do the top 3 lines do?

Reply:
It checks out 2 repos: The ReadsUtil code, as well as the custom code the Action itself uses. This is intentional.